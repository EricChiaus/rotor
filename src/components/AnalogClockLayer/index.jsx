import React, { useContext } from 'react';
import { TimeContext } from '../../contexts/TimeContext';
import AnalogClock from '../AnalogClock';

const AnalogClockLayer = () => {
    const { hours, minutes, seconds } = useContext(TimeContext);
    return <AnalogClock hours={hours} minutes={minutes} seconds={seconds} />;
};

export default AnalogClockLayer;
