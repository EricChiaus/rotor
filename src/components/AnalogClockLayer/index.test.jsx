import { render } from '@testing-library/react';
import AnalogClockLayer from '.';

describe('<AnalogClockLayer />', () => {
    it('matches the snapshot', () => {
        const { container } = render(<AnalogClockLayer />);
        expect(container).toMatchSnapshot();
    });
});
