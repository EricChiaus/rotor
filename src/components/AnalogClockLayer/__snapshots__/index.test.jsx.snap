// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AnalogClockLayer /> matches the snapshot 1`] = `
<div>
  <div
    class="analog-clock"
  >
    <div
      class="hours"
      style="transform: rotateZ(0deg;"
    />
    <div
      class="minutes"
      style="transform: rotateZ(0deg;"
    />
    <div
      class="seconds"
      style="transform: rotateZ(0deg;"
    />
  </div>
</div>
`;
